core/src/main/java/com/gamification/core/enumeration/Role.java

```diff
@@ -0,0 +1,5 @@
+package com.gamification.core.enumeration;
+
+public enum Role {
```

Comment:
in this project we are going to follow `EN_*` regex to create enum calsses.
please change enum class name to `EN_Role`